bindings/python/CMakeLists.txt

```diff
@@ -0,0 +1,85 @@
+find_package(PythonInterp REQUIRED)
+find_package(PythonLibs REQUIRED)
+
+if (NOT boost-python)
+	set(_boost_python "python-${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}")
```

Comment:
I've never seen the Boost Python libraries follow this naming convention. By default they are named `boost_python` for Python 2 and `boost_python3` for Python 3. The default should be to follow upstream's naming.

Reply:
Done